hack/propagate-api-descriptions.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Unless we're checking that they match (doubtful and probably incorrect), I'd say the verify script is sufficient to smack people.  This saves them a copy/paste at the cost of them having to know to use it.